commcare_connect/opportunity/forms.py

```diff
@@ -266,10 +269,30 @@ def __init__(self, *args, **kwargs):
                 help="Start date can't be edited if it was set in past" if self.is_start_date_readonly else None,
             ),
             Field("end_date"),
-            Field("max_users", oninput=f"id_total_budget.value = {budget_per_user} * parseInt(this.value || 0)"),
+            Field(
+                "max_users",
+                oninput=f"id_total_budget.value = ({self.budget_per_user} + {self.payment_units_max_total}"
+                f"* id_org_pay_per_visit.value || 0) * parseInt(this.value || 0)",
```

Comment:
This calculation might break the opportunity creation form for normal opportunities.

```suggestion
                f"* parseInt(document.getElementById(id_org_pay_per_visit).value || 0)) * parseInt(this.value || 0)",
```

Reply:
Thanks